package.json

```diff
@@ -64,9 +64,10 @@
     "uiFramework:createComponent": "cd packages/kbn-ui-framework && yarn createComponent",
     "uiFramework:documentComponent": "cd packages/kbn-ui-framework && yarn documentComponent",
     "kbn:watch": "node scripts/kibana --dev --logging.json=false",
+    "build:refs": "tsc -b tsconfig.refs.json",
     "build:types": "tsc --p tsconfig.types.json",
     "docs:acceptApiChanges": "node --max-old-space-size=6144 scripts/check_published_api_changes.js --accept",
-    "kbn:bootstrap": "node scripts/register_git_hook",
+    "kbn:bootstrap": "yarn build:refs && node scripts/register_git_hook",
```

Comment:
`Because dependent projects make use of .d.ts files that are built from their dependencies, you’ll either have to check in certain build outputs or build a project after cloning it before you can navigate the project in an editor without seeing spurious errors.`
from https://www.typescriptlang.org/docs/handbook/project-references.html#caveats-for-project-references